test/e2e_node/pod_resize_test.go

```diff
@@ -1295,6 +1302,570 @@ func doPodResizeErrorTests() {
 	}
 }
 
+func doConsecutivePodResizeTests() {
```

Comment:
I'm refactoring IPPR cluster and node e2e tests in https://github.com/kubernetes/kubernetes/pull/128143.

Can we wait until that PR is merged?